pyvista/plotting/plotting.py

```diff
@@ -2883,13 +2883,22 @@ def link_views(self, views=0):
             for renderer in self.renderers:
                 renderer.camera = self.renderers[views].camera
             return
+        elif issubclass(views.__class__, pyvista.plotting.BasePlotter):
```

Comment:
This is better phrased as
```suggestion
        elif isinstance(views, pyvista.BasePlotter):
```
And `BasePlotter` seems exposed at the top level, so I think we should use that for simplicity.